python/pipeline/pupil.py

```diff
@@ -1076,33 +1188,586 @@ def plot_fitting(key, start, end=-1, fit_type='Circle', fig=None, ax=None, mask_
         config['config_path'] = dlc_config['config_path']
         config['shuffle'] = dlc_config['shuffle']
         config['trainingsetindex'] = dlc_config['trainingsetindex']
-        config['cropped_coords'] =cropped_coords
+        config['cropped_coords'] = cropped_coords
 
         # find path to original video symlink
         base_path = os.path.splitext(avi_path)[0] + '_tracking'
-        video_path = os.path.join(base_path, os.path.basename((Eye() & key).get_video_path()))
+        video_path = os.path.join(
+            base_path, os.path.basename((Eye() & key).get_video_path()))
 
         config['orig_video_path'] = video_path
-        
-        pupil_fit = DLC_tools.PupilFitting(config=config, bodyparts='all', cropped=True)
+
+        pupil_fit = DLC_tools.DeeplabcutPupilFitting(
+            config=config, bodyparts='all', cropped=True)
 
         # play with these parameters for better visualization of the fitting
         pupil_fit.line_thickness = 2
         pupil_fit.fontsize = 10
-        pupil_fit.circle_color = (0,0,255)
-        pupil_fit.ellipse_color = (0,0,255)
+        pupil_fit.circle_color = (0, 0, 255)
+        pupil_fit.ellipse_color = (0, 0, 255)
         pupil_fit.dotsize = 4
 
         # check if we plot only a single frame. If so, return fig and ax
         if end == start + 1:
-            ax = pupil_fit.plot_fitted_frame(start, ax=ax, fitting_method=fit_type)
+            ax = pupil_fit.plot_fitted_frame(
+                start, ax=ax, fitting_method=fit_type)
 
             # remove mask (i.e. visible area)
             if not mask_flag:
                 del ax.images[1]
 
             return fig, ax
-        
+
         else:
-            pupil_fit.plot_fitted_multi_frames(start=start, end=end, fitting_method = fit_type)
-        
\ No newline at end of file
+            pupil_fit.plot_fitted_multi_frames(
+                start=start, end=end, fitting_method=fit_type)
+
+
+# @schema
+# class ProcessedFittedPupil(dj.Computed):
```

Comment:
these are "work in progress", if they end up not being in the latest version, remember to delete the tables from the database

Reply:
noted. I think it will be dropped altogether later on anyways, but will confirm once i am back.

Reply:
don't delete/resolve this comment so that i can remember